api/src/test/scala/hmda/api/http/institutions/SubmissionPathsSpec.scala

```diff
@@ -0,0 +1,68 @@
+package hmda.api.http.institutions
+
+import akka.event.{LoggingAdapter, NoLogging}
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
+import akka.http.scaladsl.server.MethodRejection
+import akka.util.Timeout
+import hmda.api.http.InstitutionSpec
+import hmda.api.model.{ErrorResponse, SubmissionStatusWrapper, SubmissionWrapper}
+import hmda.model.fi._
+import hmda.persistence.demo.DemoData
+
+import scala.concurrent.duration._
+
+class SubmissionPathsSpec extends InstitutionSpec {
+
+  override val log: LoggingAdapter = NoLogging
```

Comment:
Could it work to put these lines (initializing `log`, `timeout`, and `ec`) in the parent spec? If so, that would be nice, to reduce duplication.